articles/TOC.yml

```diff
@@ -1,44 +1,51 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Getting started
   items:
-  - name: Get started
-    href: welcome.md
-  - name: Five questions about quantum computing
-    items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
```

Comment:
We should add a couple of lines explaining what QDK is in overview/what-is-qsharp.md, as QDK is a term people will see first all over the site. 

![image](https://user-images.githubusercontent.com/59890648/73983449-55aae900-48eb-11ea-92ea-abe6c2e428c0.png)

Reply:
Absolutely agree, but I figure that's probably more fitting for a new/different PR---perhaps even a more robust page where the full scope/functionality of the QDK is explained (e.g. with diagrams that illustrate how the Q# language, host programs, Q# libraries, etc. all fit together, like we've discussed occasionally)

Reply:
All looking pretty good. @gillenhaalb are you able to add a paragraph briefly about what QDK is? Then we can merge it to master and live.